Cargo.toml

```diff
@@ -7,6 +7,10 @@ authors = [
   "Phil Jenvey <pjenvey@underboss.org>",
 ]
 
+[profile.release]
+# Enables line numbers in Sentry
+debug = true
```

Comment:
Took me a while to track this down, we came across it for [autopush-rs](https://github.com/mozilla-services/autopush-rs/issues/77). There's also a not well documented debug value of either 1 (or 2, but true is the equivalent of 2). This value's passed along to rustc -C debuginfo:

```
    -C            debuginfo=val -- debug info emission level, 0 = no debug info, 1 = line tables only, 2 = full debug info with variable and type information
```

`debug = 1` sounds ideal for release builds, just producing the line tables

There's been some reports of slowdowns with full release = 2 which may or may not be fixed:

https://github.com/rust-lang/rust/issues/49227

debug = 1 should at the very least reduce the binary size